jsftemplating/src/main/java/com/sun/jsftemplating/el/PageSessionResolver.java

```diff
@@ -98,6 +98,13 @@ public Class<?> getType(ELContext elContext, Object base, Object property) {
     @Override
     public void setValue(ELContext elContext, Object base, Object property, Object value) {
         checkPropertyFound(base, property);
```

Comment:
I think we shouldn't set values for base != null. Thus I suggest replace `checkPropertyFound` method with something like this:

```java
if (base != null) {
    return;
}

if (property == null) {
    throw new PropertyNotFoundException();
}
```

Reply:
Makes sense, I've updated the PR.